docs/how-to-work-on-coding-challenges.md

```diff
@@ -7,28 +7,28 @@
 
 Our goal is to develop a fun and clear interactive learning experience.
 
-Designing interactive coding challenges is difficult. It would be much easier to write a lengthy explanation or to create a video tutorial, and there's a place for those on Medium and YouTube. However, for our core curriculum, we're sticking with what works best for most people - a fully interactive, video game-like experience.
+Designing interactive coding challenges is difficult. It would be much easier to write a lengthy explanation or to create a video tutorial, and there's a place for those on Medium and YouTube. However, for our core curriculum, we're sticking with what works best for most people—a fully interactive, video game-like experience.
 
-We want campers to achieve a flow state. We want them to build momentum and blast through our curriculum with as few snags as possible. We want them to go into the projects with confidence and a wide exposure to programming concepts.
+We want campers to achieve a _flow state_. We want them to build momentum and blast through our curriculum with as few snags as possible. We want them to go into the projects with confidence, and gain a wide exposure to programming concepts.
 
-Creating these challenges requires immense creativity and attention to detail. But you'll have plenty of help. You have support from a whole team of contributors, whom you can bounce ideas off of and demo your challenges to. Stay active in the [contributors room](https://gitter.im/freecodecamp/contributors) and ask lots of questions.
+Creating these challenges requires immense creativity and attention to detail, but you'll have plenty of help. You have support from a whole team of contributors, whom you can bounce ideas off and demo your challenges to. Stay active in the [contributors room](https://gitter.im/freecodecamp/contributors) and ask lots of questions.
 
 With your help, we can design an interactive coding curriculum that will help millions of people learn to code for years to come.
 
 The content for each challenge is stored in its own markdown file. This markdown file is later coverted using our tools, to webpage that you can interact with. You can find all of freeCodeCamp.org's curricular content in the [`/curriculum`](/curriculum) directory.
 
 There are two ways you can work on these challenges:
 
-- We recommend, you clone the project and edit locally on your computer. For help with that, read our [contributing guidelines here](/CONTRIBUTING.md).
-- Or optionally, you can modify a challenge within the GitHub interface, by clicking the pencil icon to start editing the file.
+- We recommend you clone the project and edit it locally on your computer. For help with that, read our [contributing guidelines](/CONTRIBUTING.md).
+- Alternatively, you can modify a challenge within GitHub.com itself by clicking the pencil icon to start editing the file.
```

Comment:
"Alternatively" is a great word choice here, and the sentence is much clearer without the extra comma. However I do prefer some of the original wording:

`Alternatively, you can modify a challenge within the GitHub interface by clicking the pencil icon to start editing the file.`

But that's just my preference. I'd like to hear what you and another reviewer think.

Reply:
Totally cool with keeping that phrasing.